product/views.py

```diff
@@ -1,3 +1,76 @@
-from django.shortcuts import render
+import json
+import decimal
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+ 
+from .models      import  (
+    Category,
+    Subcategory, 
+    Product, 
+    ProductImage, 
+    Discount, 
+    Review
+)
+
+from user.models  import Member #login_decorator
+
+
+class ProductListView(View):
+    def get(self, request):
+            category_seq    = request.GET.get('category', None)
+            subcategory_seq = request.GET.get('subcategory', None)
+            sort            = request.GET.get('sort', None)
+
+            if subcategory_seq == None:
+                products = Product.objects.filter(category=category_seq).order_by(sort).values()
+            else:
+                products = Product.objects.filter(category=category_seq, subcategory=subcategory_seq).order_by(sort).values()
```

Comment:
이 경우는 Q객체를 쓰면 해결될것 같습니다.
```suggestion
                products = Product.objects.filter(Q(category=category_seq) | Q (subcategory=subcategory_seq)).order_by(sort).values()
```